### ISSUE TITLE
App wont open again after using it.

### ISSUE THREAD

**Issue by Longshoez** [Genymobile/scrcpy#3077]
- [ ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ ] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: [Windows 10]
 - scrcpy version: [e.g. 1.23]
 - installation method: [Windows release]
 - device model: Samsung galaxy A7 20218
 - Android version: [10]

**Describe the bug**
Upon closing the app once, it wont open again when needed, it just shows the terminal for a second and it closes. 

I couldnt catch the error because the terminal closes as soon as it opens. but i have encountered that i can run it again by restarting my pc.

```
no terminal 
```

**Comment by rom1v** [comment]
> I couldnt catch the error because the terminal closes as soon as it open

Run `scrcpy-console.bat`.

**Comment by Longshoez** [comment]
yeah i know, i read the FAQ, since i couldnt do it on the app terminal bc it crashes, i did it on the cmd and powershell. still no luck

> scrcpy-console.bat

![image](https://user-images.githubusercontent.com/26613957/156670964-75c100bb-2e98-4dd4-b713-d35944f68149.png)

**Comment by rom1v** [comment]
From the scrcpy directory, double-click on `scrcpy-console.bat`.
